content/docs/user-guide/gems/reference/aws/aws-core/resource-mapping-files.md

```diff
@@ -47,17 +51,17 @@ The mapping file also supports three global attributes:
 }
 ```
 
-This mappings file has a single mapping: **MyKey**, which is a Lambda function called "ExampleLambda". It is deployed in us-west-2 in the AWS account with account ID "123456789123".
+This mapping file has a single mapping: **MyKey**, which is a Lambda function called "ExampleLambda". It is deployed in us-west-2 in the AWS account with account ID "123456789123".
 
 ## Generating a resource mappings file
 
-The mapping files have a well-defined JSON format. They can be generated using custom tools or by hand. However, the easiest way to generated these files is by using the visual [resource mapping tool](/docs/user-guide/gems/reference/aws/aws-core/resource-mapping-tool.md) provided with O3DE to generate and edit the files.
+The mapping files have a well-defined [JSON format](/docs/user-guide/gems/reference/aws/aws-core/resource-mapping-schema.md). They can be generated using custom tools or by hand. However, the easiest way to generate or edit these files is by using the visual [resource mapping tool](/docs/user-guide/gems/reference/aws/aws-core/resource-mapping-tool.md) provided with O3DE.
 
 All mapping files should be saved into the project's `Config` directory.
 
 ## Using resource mappings
 
-The resource mapping file to use on load is controlled by the `ResourceMappingConfigFileName` in the `awscoreconfiguration.setreg` file, located in the project's `Registry` directory. See [Project Settings](./getting-started.md#project-settings) for more information about this file. The AWS Core Gem will read this mapping file and support access to the mappings.
+The resource mapping file to use on load is controlled by the `ResourceMappingConfigFileName` setting in the `awscoreconfiguration.setreg` file; located in the project's `Registry` directory. See [Project Settings](./getting-started.md#project-settings) for more information about this file. The AWS Core Gem will read this mapping file and support access to the mappings.
```

Comment:
Comma instead of semicolon, because the phrase after it is not a complete sentence.
```suggestion
The resource mapping file to use on load is controlled by the `ResourceMappingConfigFileName` setting in the `awscoreconfiguration.setreg` file, located in the project's `Registry` directory. See [Project Settings](./getting-started.md#project-settings) for more information about this file. The AWS Core Gem will read this mapping file and support access to the mappings.
```

Reply:
Fixed.